### PR TITLE
AI-driven alerting (aka "smart alerts") -  GA

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request:
- Removes the beta tagging from this article:
https://help.sumologic.com/docs/alerts/monitors/use-playbooks-with-monitors/
- Adds a release note:
https://help.sumologic.com/release-notes-service/2024/03/12/alerts/
- Updates these articles:
  - https://help.sumologic.com/docs/alerts/monitors/create-monitor/
  - https://help.sumologic.com/docs/alerts/monitors/alert-response/
  - https://help.sumologic.com/docs/alerts/monitors/alert-grouping/

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

### CODE REVIEW

**Comment by jpipkin1** [comment]
@banant, could you review this PR?

File: docs/alerts/monitors/create-monitor.md [hunk 1]
```diff
@@ -326,24 +318,31 @@ To add notifications, click the **Add Notification** button. You can add more
    * If your connection type is Lambda, Microsoft Teams, OpsGenie, PagerDuty, Slack, or a generic webhook, the **Recovery** checkbox enables an automatic resolution process that updates the connection when an alert has recovered within Sumo Logic. Support for other connection types is coming soon.
    * **Add Notifications** to add additional notification channels as needed. You can configure different notifications for each trigger type, critical, warning, and missing data.
 
-## Step 4: Monitor details
+## Step 4: Playbook (optional)
+
+The fourth step when you create a new monitor is to add playbooks.
+
+<img src={useBaseUrl('img/monitors/new-monitor-playbook.png')} alt="
+Playbook" style={{border: '1px solid black'}} width="800"/>
```
Comment (kimsauce) [inline docs/alerts/monitors/create-monitor.md:326]:
```suggestion
<img src={useBaseUrl('img/monitors/new-monitor-playbook.png')} alt="Playbook" style={{border: '1px solid black'}} width="800"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 2]
```diff
@@ -326,24 +318,31 @@ To add notifications, click the **Add Notification** button. You can add more
...
+
+In this step you can add a **Playbook** to run in response to an alert.
```
Comment (kimsauce) [inline docs/alerts/monitors/create-monitor.md:328]:
```suggestion
In this step, you can add a **Playbook** to run in response to an alert. 
```

File: docs/alerts/monitors/use-playbooks-with-monitors.md [hunk 3]
```diff
@@ -40,21 +35,18 @@ For more information, see [Add an automated playbook to a monitor](/docs/alerts/
...
-1.  (Optional) Click **Manage Playbooks** to open playbooks previously authored in the Automation Service. From here, you can view the selected playbook and look at its actions to determine if they are the ones you want to run when the alert is triggered. You can also edit the playbook, or if you want, create a new playbook to use in the monitor.  
-1. Click **Save** to save the monitor settings.
+1. In [Step 4: Playbook](/docs/alerts/monitors/create-monitor/#step-4-playbook-optional), click **Automated Playbooks** to select a [playbook in the Automation Service](/docs/platform-services/automation-service/automation-service-playbooks/) to run when an alert is fired. <br/><img src={useBaseUrl('img/monitors/monitor-playbooks-ui.png')} alt="Add a playbook to a monitor" style={{border: '1px solid black'}} width="800" />
+1.  (Optional) Click **Manage Playbooks** to see playbooks in the Automation Service. From here you can view the playbook you selected above and look at its actions to determine if they are the ones you want to run when the alert is triggered. You can also edit the playbook, or if you want, create a new playbook to use in the monitor.
```
Comment (kimsauce) [inline docs/alerts/monitors/use-playbooks-with-monitors.md:45]:
```suggestion
1. (Optional) Click **Manage Playbooks** to see playbooks in the Automation Service. From here you can view the playbook you selected above and look at its actions to determine if they are the ones you want to run when the alert is triggered. You can also edit the playbook, or if you want, create a new playbook to use in the monitor.  
```

**Review by kimsauce — APPROVED** [review]
approved contingent on minor fixes

File: docs/alerts/monitors/alert-grouping.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ Alert Grouping works for both Logs and Metrics Monitors.
...
    * **One alert per monitor**: Choose this option if you want to only receive a single alert for the entire monitor.
-   * **One alert per [group]**: Allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, you would receive one alert for each `service` that has error count greater than 50. The input field has an auto-completion dropdown that allows you to select all the applicable fields from your query.<br/><img src={useBaseUrl('img/monitors/setup-logs.png')} alt="alert-grouping" />
+   * **One alert per [group]**: Allows you to receive one notification per each unique value of the grouping field(s). You can pick more than one field for the grouping condition. In the example below, you would receive one alert for each `service` that has error count greater than 50. The input field has an auto-completion dropdown that allows you to select all the applicable fields from your query.<br/><img src={useBaseUrl('img/monitors/setup-logs.png')} alt="alert-grouping" style={{border: '1px solid black'}} width="800" />
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-grouping.md:40]: The `style={{border: '1px solid black'}}` just need to be changed to `style={{border: '1px solid gray'}}`. You could do a global find all > replace in VS Code.

**Review by kimsauce — CHANGES_REQUESTED** [review]
One small change! Image border color.

File: docs/alerts/monitors/alert-grouping.md (hunk 4 above)
Comment (jpipkin1) [reply docs/alerts/monitors/alert-grouping.md:40]: Good catch, @kimsauce. Done. See commit "[Add gray borders to images](https://github.com/SumoLogic/sumologic-documentation/pull/3397/commits/e7a5b4069fe5d2a865d1069381bb424d13b0884a)".

Comment (kimsauce) [reply docs/alerts/monitors/alert-grouping.md]: thank you! For some reason it looks like the black wasn't replaced with gray. I took care of it [here](https://github.com/SumoLogic/sumologic-documentation/pull/3397/commits/8f5a0ae3dc525100036517d37d935806c0a52f8d)!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**